composer.json

```diff
@@ -11,18 +11,17 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 | ^8.0",
         "ext-json": "*"
     },
 
     "require-dev": {
-        "roave/security-advisories": "dev-master",
-        "roave/backward-compatibility-check": "^2.0",
-        "codeception/codeception": "^2.5",
-        "php-coveralls/php-coveralls": "^2.1",
-        "squizlabs/php_codesniffer": "^3.3",
+        "codeception/codeception": "^4.1",
+        "squizlabs/php_codesniffer": "^3.5",
         "predis/predis": "^1.1",
-        "ext-redis": "*"
+        "ext-redis": "*",
+        "codeception/module-asserts": "^1.3",
+        "codeception/assert-throws": "^1.1"
```

Comment:
I think we dont neeed assert-throws. module asserts will be enough.

Reply:
Ok, my mistake. I always use phpunit, may be I forgot that API is the same.